src/theme/_typography.scss

```diff
@@ -374,4 +374,58 @@ $font-path: '/assets/fonts/' !default;
 
 body {
   font-family: "Montserrat", sans-serif;
+  font-size: $font-size-l;
+}
+& {
+  font-family: “Montserrat”, sans-serif !important;
```

Comment:
enkel &{ gebruiken => body gaat niet werken wanneer dit geimporteerd wordt in june-app.

Reply:
En zelfs voor font-family bestaat er al een variabele die je kan hergebruiken :)